src/applications/fry-dea/config/form.js

```diff
@@ -527,6 +527,143 @@ const formConfig = {
+          title: 'Marriage date',
+          path: 'new/additional/consideration/marriage/date',
+          uiSchema: {
+            'view:subHeadings': {
+              'ui:description': (
+                <>
+                  <h3>Marriage date</h3>
+                </>
+              ),
+            },
+            [newFormFields.newAdditionalConsiderations.newMarriageDate]: {
+              ...currentOrPastDateUI(
+                'When did you get married to your chosen Veteran or service member?',
+              ),
+            },
+          },
+          schema: {
+            type: 'object',
+            required: [
+              newFormFields.newAdditionalConsiderations.newMarriageDate,
+            ],
+            properties: {
+              'view:subHeadings': {
+                type: 'object',
+                properties: {},
+              },
+              [newFormFields.newAdditionalConsiderations.newMarriageDate]: date,
+            },
+          },
+        },
+        [newFormPages.newAdditionalConsiderations.newMarriageInformation]: {
+          title: 'Marriage information',
+          path: 'new/additional/consideration/marriage/information',
+          uiSchema: {
+            'view:subHeadings': {
+              'ui:description': (
+                <>
+                  <h3>Marriage information</h3>
+                </>
+              ),
+            },
+            [newFormFields.newAdditionalConsiderations
+              .newMarriageInformation]: {
+              'ui:title':
+                'What’s the status of your marriage with your chosen Veteran or service member?',
+              'ui:widget': 'radio',
+            },
+          },
+          schema: {
+            type: 'object',
+            required: [
+              newFormFields.newAdditionalConsiderations.newMarriageInformation,
+            ],
+            properties: {
+              'view:subHeadings': {
+                type: 'object',
+                properties: {},
+              },
+              [newFormFields.newAdditionalConsiderations
+                .newMarriageInformation]: {
+                type: 'string',
+                enum: [
+                  'Married',
+                  'Divorced (or a divorce is in progress)',
+                  'Marriage was annulled (or annulment is in progress)',
+                  'Widowed',
+                ],
+              },
+            },
+          },
+        },
+        [newFormPages.newAdditionalConsiderations.newRemarriage]: {
+          title: 'Remarriage',
+          path: 'new/additional/consideration/remarriage/information',
+          uiSchema: {
+            'view:subHeadings': {
+              'ui:description': (
+                <>
+                  <h3>Remarriage</h3>
+                </>
+              ),
+            },
+            [newFormFields.newAdditionalConsiderations.newRemarriage]: {
+              'ui:title': 'Have you been remarried since your divorce?',
```

Comment:
There are actually 3 versions of the remarriage page: one each for divorced, annulled, and widowed. I'm guessing you should be able to keep just this page and just change the title.